package.json

```diff
@@ -19,11 +19,11 @@
     },
     "license": "MIT",
     "dependencies": {
-        "leaflet": "^0.7.7",
+        "leaflet": "^1.6.0",
```

Comment:
Du coup, ça empêche d'utiliser Leaflet.Snap avec toute version < 1.6.0 alors qu'à priori ça marche. Est-ce qu'il ne vaut pas mieux mettre ">=0.7.7" ou ">=1.0.0" ?

D'ailleurs, est-ce que ce n'est pas plutôt aller dans peerDependencies ? Cf. https://lexi-lambda.github.io/blog/2016/08/24/understanding-the-npm-dependency-model/